docs/Moderation.md

```diff
@@ -26,8 +26,10 @@ Moderation mode is enabled by clicking ![ModModeDisabled](./images/moderation/mo
 | pajbot2 report | `/w botname #{channel.name} !report {user.name} being rude` |
 | pajbot2 longreport | `/w botname #{channel.name} !longreport {user.name} being very rude` |
 | Open the user's usercard | `/user {user.name}` |
+| pajbot2 banphrase | `!add banphrase {message}`² |
```

Comment:
```suggestion
| pajbot banphrase | `/w botname !add banphrase {message}`² |
```
pb1 probably a better example, and whispers since the phrase in question could be ToS